### PR TITLE
change in data center location for ASC tenants

### PR DESCRIPTION
**Pull request by malidgi**
in the past (until a few months ago), is MDE was enabled for the first time via the integration with Azure defender, the data location for MDE data was hardcoded EU. 
we changed it a few months ago, and today we use the same methodology to choose the data center location as all other tenants. 
contact me for more details

### CODE REVIEW

**Comment by yogkumgit** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks

File: microsoft-365/security/defender/m365d-enable.md [hunk 1]
```diff
@@ -60,7 +60,7 @@ Microsoft 365 Defender will store and process data in the [same location used by
 Select **Need help?** in the Microsoft 365 security center to contact Microsoft support about provisioning Microsoft 365 Defender in a different data center location.
 
 > [!NOTE]
-> Microsoft Defender for Endpoint automatically provisions in European Union (EU) data centers when turned on through Azure Defender. Microsoft 365 Defender will automatically provision in the same EU data center for customers who have provisioned Defender for Endpoint in this manner.
+> in the past, Microsoft Defender for Endpoint automatically used to be provisioned European Union (EU) data centers when turned on through Azure Defender. Microsoft 365 Defender will automatically provision in the same EU data center for customers who have provisioned Defender for Endpoint in this manner.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender/m365d-enable.md]:
```suggestion
> In the past, Microsoft Defender for Endpoint automatically provisioned in European Union (EU) data centers when turned on through Azure Defender. Microsoft 365 Defender will automatically provision in the same EU data center for customers who have provisioned Defender for Endpoint in this manner in the past.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by yogkumgit** [comment]
@malidgi Please implement the suggestions. Thanks

**Comment by malidgi** [comment]
@yogkumgit can you be more specific about the required action on my side?

**Comment by yogkumgit** [comment]
@malidgi Johan performed copy edit and have suggested some changes. :) Thanks

**Comment by malidgi** [comment]
@yogkumgit the changes look good!

**Comment by JohanFreelancer9** [comment]
@malidgi - please click on the commit button to commit my changes. Thanks so much.

**Comment by yogkumgit** [comment]
@JohanFreelancer9

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@Dansimp Please merge this PR. Thanks.

**Pull request merged**